core/fake/domain_storage.go

```diff
@@ -18,38 +18,38 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/google/keytransparency/core/adminstorage"
+	"github.com/google/keytransparency/core/domainstorage"
 )
 
 // AdminStorage implements adminstorage.Storage
 type AdminStorage struct {
-	domains map[string]*adminstorage.Domain
+	domains map[string]*domainstorage.Domain
 }
 
 // NewAdminStorage returns a fake adminstorage.Storage
 func NewAdminStorage() *AdminStorage {
```

Comment:
And `NewAdminStorage` -> `NewDomainStorage`

Reply:
Fixed.